encoding.bs

```diff
@@ -1249,10 +1404,14 @@ method, when invoked, must run these steps:
  Exposed=(Window,Worker)]
 interface TextEncoder {
   readonly attribute DOMString encoding;
+  [SameObject] readonly attribute ReadableStream readable;
+  [SameObject] readonly attribute WritableStream writable;
   [NewObject] Uint8Array encode(optional USVString input = "");
 };</pre>
 
-<p>A {{TextEncoder}} object has an associated <dfn for=TextEncoder>encoder</dfn>.
+<p>A {{TextEncoder}} object has an associated <dfn for=TextEncoder>encoder</dfn>, <dfn
+for=TextEncoder>transform</dfn> (a {{TransformStream}} object) and <dfn for=TextEncoder>pending high
+surrogate</dfn> (initially unset).
```

Comment:
No wrapping inside phrasing elements. This also has the same problem where types and initial values are mixed. I recommend omitting types for now and filing a follow-up issue.